packages/fxa-settings/src/components/Settings/BentoMenu/en.ftl

```diff
@@ -1,7 +1,9 @@
 # BentoMenu component
 
 bento-menu-title = { -brand-firefox } Bento Menu
+bento-menu-title-2 = { -brand-mozilla } Bento Menu
 bento-menu-firefox-title = { -brand-firefox } is tech that fights for your online privacy.
+bento-menu-mozilla-title = { -brand-mozilla } is tech that fights for your online privacy.
 
 bento-menu-vpn-2 = { -product-mozilla-vpn }
 bento-menu-monitor-2 = { -product-firefox-monitor }
```

Comment:
We are not updating other brand term references yet, like `Firefox Monitor` / `Firefox Relay`.